options.go

```diff
@@ -13,6 +13,13 @@ type Options struct {
 	// Print step definitions found and exit
 	ShowStepDefinitions bool
 
+	// Run scenarios in random order.
+	//
+	// This is especially helpful for detecting situations
+	// where you have state leaking between scenarios, which
+	// can cause flickering or fragile tests.
+	RandomOrder bool
```

Comment:
maybe could be `int` where `0` is not a random (should be explained in the comment and flag). Then this could be also available in flags. And you would not need to print it in suite, but instead if you run it through flags - you know it already, if from TestMain - you can print it there. Because godog might export junit xml to stdout if different format is used.